src/AccessControl/rolemanager.py

```diff
@@ -295,7 +295,7 @@ def users_with_local_role(self, role):
         for user, roles in self.get_local_roles():
             if role in roles:
                 got[user] = 1
-        return got.keys()
+        return list(got.keys())
```

Comment:
```suggestion
        return tuple(got.keys())
```

Why is having a list here important? I can see how code might stumble over the changed return type from the `keys` method, which returns a dictionary view in Python 3, but if you make a change I'd rather see it cast as a tuple instead of a list. That highlights the fact that the piece of data that it returns is "read only"

Reply:
Thanks for your quick reply. In our code, for some reasons, we get the first user returned and thus do `user_with_local_role()[0]` which raises a TypeError in Python3.

But this change was more about keeping the same API on both Python versions.

Good point for the tuple, I have changed that then and made the commit message clearer.